selfdrive/car/subaru/carcontroller.py

```diff
@@ -35,10 +41,18 @@ def update(self, CC, CS, now_nanos):
       if not CC.latActive:
         apply_steer = 0
 
+      apply_steer_req = CC.latActive
+
+      if self.CP.carFingerprint in STEER_RATE_LIMITED:
+        # Steering rate fault prevention
+        self.steer_rate_counter, apply_steer_req = \
+          common_fault_avoidance(abs(CS.out.steeringRateDeg) > MAX_STEER_RATE, apply_steer_req,
+                                 self.steer_rate_counter, MAX_STEER_RATE_FRAMES)
```

Comment:
do any pre-globals use the steer fault prevention?

Reply:
not current, I can move it under the non-preglobal section